app/views/admin/schools/show.html.erb

```diff
@@ -1,169 +1,21 @@
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Administrative District Name")
-      row.with_value(text: @school.administrative_district_name)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Administrative District Name")
-      row.with_value(text: @school.administrative_district_name)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Establishment Number")
-      row.with_value(text: @school.establishment_number)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Local Authority Code")
-      row.with_value(text: @school.local_authority_code)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Local Authority Name")
-      row.with_value(text: @school.local_authority_name)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Phase Name")
-      row.with_value(text: @school.phase_name)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Funding Eligibility")
-      row.with_value(text: @school.funding_eligibility)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Induction Eligibility")
-      row.with_value(text: @school.induction_eligibility)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "In England")
-      row.with_value(text: @school.in_england)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Section 41 Approved")
-      row.with_value(text: @school.section_41_approved)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "UKPRN")
-      row.with_value(text: @school.ukprn)
-    end
-
-    sl.with_row do |row|
-      row.with_key(text: "Website")
-      row.with_value(text: @school.website)
-    end
-  end
-%>
-
-<h2 class="govuk-heading-m">Early Career Teachers</h2>
-
-<%= govuk_table do |table|
-  table.with_head do |head|
-    head.with_row do |row|
-      row.with_cell(text: "Name")
-      row.with_cell(text: "TRN")
-    end
-  end
-
-  table.with_body do |body|
-    @school.ect_teachers.each do |teacher|
-      body.with_row do |row|
-        row.with_cell do
-          govuk_link_to teacher_full_name(teacher), admin_teacher_path(teacher)
-        end
-        row.with_cell(text: teacher.trn)
-      end
-    end
-  end
-end %>
-
-<h2 class="govuk-heading-m">Mentors</h2>
-
-<%= govuk_table do |table|
-  table.with_head do |head|
-    head.with_row do |row|
-      row.with_cell(text: "Name")
-      row.with_cell(text: "TRN")
-    end
-  end
-
-  table.with_body do |body|
-    @school.mentor_teachers.each do |teacher|
-      body.with_row do |row|
-        row.with_cell do
-          govuk_link_to teacher_full_name(teacher), admin_teacher_path(teacher)
-        end
-        row.with_cell(text: teacher.trn)
-      end
-    end
-  end
-end %>
+<% page_data(
+  title: @school.name,
+  caption: "URN: #{@school.urn}",
+  caption_size: "xl",
+  header_size: "xl",
+  breadcrumbs: @breadcrumbs
+) %>
+
+<%= govuk_tabs(title: nil) do |tabs| %>
```

Comment:
Actually I just realised that the design doesn't have tabs here, it uses the secondary nav pattern as per this screenshot. I think we're best avoiding tabs if pos and using it (splitting the below bits onto three separate pages)

<img width="1957" height="706" alt="image" src="https://github.com/user-attachments/assets/92e6883f-0542-4e51-8432-9c614704f93f" />